src/ashley/editor/decorators.py

```diff
@@ -0,0 +1,33 @@
+"""This module contains custom decorators for draftjs_exporter."""
+import logging
+
+from draftjs_exporter.dom import DOM
+
+logger = logging.getLogger("ashley.editor.decorators")
```

Comment:
It is usually recommended to follow the module name convention instead:

```suggestion
logger = logging.getLogger(__name__)
```

From the standard library documentation:

> The logger name hierarchy is analogous to the Python package hierarchy, and identical to it if you organise your loggers on a per-module basis using the recommended construction `logging.getLogger(__name__)`. That’s because in a module, `__name__` is the module’s name in the Python package namespace.

Reference: https://docs.python.org/3/library/logging.html

Reply:
The same remark applies everywhere.

Reply:
Thanks for this tip !
I fixed the logger introduced by this PR and i'll fix the others in another PR.